primitives/npos-elections/src/phragmen.rs

```diff
@@ -63,6 +63,9 @@ const DEN: ExtendedBalance = ExtendedBalance::max_value();
 /// `expect` this to return `Ok`.
 ///
 /// This can only fail if the normalization fails.
+///
+/// Note that rounding errors can potentially cause the output of this function to fail a PJR check.
```

Comment:
```suggestion
/// Note that rounding errors can potentially cause the output of this function to fail a t-PJR check where t is ....
```

better to specify which PJR test exactly

Reply:
Agree: 9c7fb95c7